src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -120,7 +120,7 @@ public interface IChartEditorPresenter : IPresenter<IChartEditorView>,
       /// <summary>
       ///    Adds a curves to the chart for all dataColumns in <paramref name="dataColumnList" /> if they do not already exist.
       ///    Curve options will be applied if they are specified and a new curve is created.All the created curves will have
-      ///    the same color. 
+      ///    the same color.
```

Comment:
@georgeDaskalakis Please review my changes and if you agree please merge.
I would love if you could explain the algorithm IN the code for grouping as we discussed on the call yesterday because it is not obvious